src/hook/api/tak/useFetchZipFile.ts

```diff
@@ -25,7 +26,7 @@ async function fetchZipFile(): Promise<{ blob: Blob; filename: string }> {
   }
 
   const data = (await res.json()) as FilesResponse;
-  const fileData = data.files.tak[0]; // Assuming you want the first file in the 'tak' array
+  const fileData = data.files.tak[getOperatingSystem() === "iOS" ? 1 : 0]; // Use the second file, which should be 'itak.zip', if user OS is "iOS"
```

Comment:
this detection should be overridable, if the detection doenst work

Reply:
done with the downloadtakzipmodal